src/hooks/use-suggestions.ts

```diff
@@ -402,7 +402,7 @@ const getError = (
 
 const isValidByDataType = (
     str: string | string[],
-    dataType: string | string[],
+    dataType: string | any[],
```

Comment:
why any[] ?

Reply:
well because of [this](https://github.com/dataloop-ai/components/pull/740/files#diff-1ead159cbb5550124194e698592bfd3b4f8963b5bfb41e5eef9e881481a9e2d9R143)

Reply:
i still dont understand. we can easily determine the type needed without the need to use any
Can you change it to the type its supposed to accept?

Reply:
@fadiDL because rotem wanted to put a dictionary into scheme which had string[] before - now it is not not an array of strings but an array of strings or objects. do you want me to type it as (string | object)[] instead of any[] here?

Reply:
> @fadiDL because rotem wanted to put a dictionary into scheme which had string[] before - now it is not not an array of strings but an array of strings or objects. do you want me to type it as (string | object)[] instead of any[] here?

type it as 

(string | {[key: string]: any})[]